lib/chef/provider/git.rb

```diff
@@ -105,7 +105,7 @@ def action_sync
       end
 
       def git_minor_version
-        @git_minor_version ||= Gem::Version.new(shell_out!("git --version", run_options).stdout.split.last)
+        @git_minor_version ||= Gem::Version.new((run_git_command ["--version"]).stdout.split.last)
```

Comment:
and we obviously often leave parens off when there's no argument -- so we don't use `.stdout.split().last()` or whatever...

dsl methods are another good case where we obviously don't bother with parens:

```ruby
file("/tmp/whatever") do
  owner("root")
  group("root")
  mode("0644")
end
```

nobody wants to type that...